src/components/yield/YieldPushFeeV3.tsx

```diff
@@ -723,7 +723,9 @@ const YieldPushFeeV3 = ({ userDataPush, getUserDataPush, PUSHPoolstats, getPUSHP
                 <Button
                   variant="outline"
                   size="medium"
-                  block
+                  css={css`
+                    flex: 1;
+                  `}
```

Comment:
```suggestion
             <Box display="flex" width="-webkit-fill-available">
              <Button
                  variant="outline"
                  size="medium"
                  block
                  >
                  ...
               </Button>
             </Box>
```

@wolfcito please follow this approach.

This can import Box component like this:

`import { Box } from "blocks"`

Reply:
Super cool @rohitmalhotra1420 , that component was exactly what was needed. I’ve made the adjustment so you can check it out